c++/src/kj/exception.h

```diff
@@ -133,6 +134,25 @@ class Exception {
   KJ_NOINLINE void addTraceHere();
   // Adds the location that called this method to the stack trace.
 
+  using DetailTypeId = unsigned long long;
```

Comment:
Looking at other kj headers, I guess `unsigned long long` is preferred to `uint_64t`, maybe because it doesn't need to pull in additional headers and is probably equivalent on any existing platform.

Reply:
Yeah, I didn't want to pull in stdint.h here, given this header is very widely included.